src/amplitude-wrapper.js

```diff
@@ -159,7 +159,7 @@ import { version } from '../package.json';
 
       // Pick the first argument as the instance name
       var name = args.shift();
-      var client = a.amplitude._iq[name];
+      var client = name === "$default_instance" ? a.amplitude : a.amplitude._iq[name];
```

Comment:
I'd prefer that we remove the default value coming from the template then handle it like this here

```suggestion
      var client = name ? a.amplitude : a.amplitude._iq[name];
```